### PR TITLE
fix(ci): check for adityapk00/lightwalletd behaviour in test harness

### PR DESCRIPTION
**Pull request by teor2345**
## Motivation

The `main` branch is failing, because the lightwalletd tests fail.

## Solution

- check for adityapk00/lightwalletd behaviour in test harness
- tidy up and document CI trigger paths

## Review

This is an urgent fix, to stop other PRs failing.

### Reviewer Checklist

  - [ ] CI passes
  - [ ] Tests for Expected Behaviour
  - [ ] Tests for Errors

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,7 +15,6 @@ on:
       - '**/Cargo.lock'
       - '**/deny.toml'
       # workflow definitions
-      - 'docker/**'
```
Comment (dconnolly) [inline .github/workflows/ci.yml:18]: ?

**Review by dconnolly — DISMISSED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (dconnolly) [reply .github/workflows/ci.yml:18]: I guess these are not used in `ci.yml`

Comment (teor2345) [reply .github/workflows/ci.yml:18]: Yep, just making it match.

**Review by dconnolly — DISMISSED** [review]

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -95,6 +95,7 @@ jobs:
             ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:${{ env.GITHUB_EVENT_PULL_REQUEST_HEAD_SHA_SHORT || env.GITHUB_SHA_SHORT }}
             ${{ env.GCR_BASE }}/${{ env.GITHUB_REPOSITORY_SLUG_URL }}/${{ env.IMAGE_NAME }}:latest
             ${{ env.GCR_BASE }}/${{ env.GITHUB_REPOSITORY_SLUG_URL }}/${{ env.IMAGE_NAME }}:${{ env.GITHUB_EVENT_PULL_REQUEST_HEAD_SHA_SHORT || env.GITHUB_SHA_SHORT }}
+          labels: ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:${{ env.GITHUB_EVENT_PULL_REQUEST_HEAD_SHA_SHORT || env.GITHUB_SHA_SHORT }}
```
Comment (teor2345) [inline .github/workflows/test.yml]: This change doesn't seem to fix the issue
```suggestion
```

**Review by gustavovalverde — APPROVED** [review]
As @dconnolly previously approved this. I'm just re-approving it.

**Pull request merged**